packages/terra-functional-testing/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@cerner/terra-functional-testing",
```

Comment:
make this package private until we are ready to release it.

Reply:
Marked as private: https://github.com/cerner/terra-toolkit/pull/403/commits/6b3fc2df5f64d37f8d470d4df000a1718ad93051